kratos/utilities/auxiliar_model_part_utilities.cpp

```diff
@@ -104,20 +105,21 @@ void AuxiliarModelPartUtilities::RemoveElementAndBelongings(
     IndexType ElementId, Flags IdentifierFlag, IndexType ThisIndex)
 {
     auto& r_array_nodes = mrModelPart.Nodes(ThisIndex);
-    #pragma omp parallel for
-    for(int i=0; i<static_cast<int>(r_array_nodes.size()); ++i) {
-        auto it_node = r_array_nodes.begin() + i;
-        it_node->Set(IdentifierFlag, true);
-    }
 
-    // TODO: Add OMP
-    for (auto& elem : mrModelPart.Elements(ThisIndex)) {
-        if (elem.Id() != ElementId) {
-            for (auto& node : elem.GetGeometry()) {
-                node.Set(IdentifierFlag, false);
-            }
+    block_for_each( 
```

Comment:
here you can actually use this method: https://github.com/KratosMultiphysics/Kratos/blob/master/kratos/utilities/variable_utils.h#L815

Reply:
As far as I understand, I have to construct an instance of ```VariableUtils``` to call that. Is there any reason for that function to be non-static? It doesn't seem to use any members.

Reply:
yes exactly
there are no reasons except legacy :)